Source/WebCore/css/CSSPseudoSelectors.json

```diff
@@ -508,6 +508,7 @@
         "before": {
             "supports-single-colon-for-compatibility": true
         },
+        "checkmark": {},
```

Comment:
We should probably put this behind a feature flag. Maybe the `appearance: base` one or a separate one, no strong preference.